src/redux/rocket/rocket.js

```diff
@@ -0,0 +1,61 @@
+const STORE_ROCKET_DATA = 'space-travelers-hub/rocket/STORE_ROCKET_DATA';
+const RESERVE_ROCKET_TICKET = 'space-travelers-hub/rocket/RESERVER_ROCKET_TICKET';
+const CANCEL_ROCKET_TICKET = 'space-travelers-hub/rocket/CANCEL_ROCKET_TICKET';
+
+const defaultState = [];
+
+const rocketReducer = (state = defaultState, action) => {
+  switch (action.type) {
+    case STORE_ROCKET_DATA:
+      return action.payload.map((item) => {
+        const {
+          rocket_id: id,
+          rocket_type: type,
+          rocket_name: name,
+          flickr_images: flickrImages,
+          description,
+          wikipedia,
+        } = item;
+        return {
+          id, type, name, flickrImages, description, wikipedia,
+        };
+      });
+    case RESERVE_ROCKET_TICKET:
+      return state.map((rocket) => {
+        if (rocket.id !== action.id) return rocket;
+        return { ...rocket, reserved: true };
+      });
+
+    case CANCEL_ROCKET_TICKET:
+      return state.map((rocket) => {
+        if (rocket.id !== action.id) return rocket;
+        return { ...rocket, reserved: false };
+      });
```

Comment:
- [ ] When we cancel a rocket, the store should be done by returning a new state object. Please, add this functionality. 🌵 
- The `reserved: false` it's ok if remains so, good job on that!